### ISSUE TITLE
ltex fails to update configuration

### ISSUE THREAD

**Issue by 00sapo** [williamboman/nvim-lsp-installer#209]
When trying to add false positives or disable a rule in ltex, it returns the error:
`E5108: Error executing lua ...taller/lua/nvim-lsp-installer/servers/ltex/configure.lua:14: attempt to index local 'f' (a nil value)`

I'm using LunarVim, but I don't think it affects the configuration, I'm using `require'lvim.lsp.manager'.setup("ltex", {})`, that should just load the default config from `lspconfig`

**Comment by abalmos** [comment]
It was removed in preparation of new code, but it appears to never have been finished. It was inteded to go into https://github.com/brymer-meneses/grammar-guard.nvim (https://github.com/brymer-meneses/grammar-guard.nvim/issues/4) and @mjlbach indicated that is the best path forward: https://github.com/neovim/nvim-lspconfig/pull/863#issuecomment-928004061 and https://github.com/brymer-meneses/grammar-guard.nvim/issues/5#issuecomment-954809067

@lbiaggi was working on backing porting the local command handlers to the new 5.1 API. This handles things like marking false positives, etc. (the code that was removed from the lspconfig PR). He may have run out of time to work on it (and maybe is using the coc plugin).

The code in this repo _works_ with a small change

```patch
diff --git a/lua/nvim-lsp-installer/servers/ltex/configure.lua b/lua/nvim-lsp-installer/servers/ltex/configure.lua
@@ -126,9 +126,9 @@ configs.ltex = {
         root_dir = function(filename)
             return util.path.dirname(filename)
         end,
-        dictionary_files = { ["en"] = { vim.fn.getcwd() .. "dictionary.ltex" } },
-        disabledrules_files = { ["en"] = { vim.fn.getcwd() .. "disable.ltex" } },
-        falsepositive_files = { ["en"] = { vim.fn.getcwd() .. "false.ltex" } },
+        dictionary_files = { ["en"] = { vim.fn.getcwd() .. "/dictionary.ltex" } },
+        disabledrules_files = { ["en"] = { vim.fn.getcwd() .. "/disable.ltex" } },
+        falsepositive_files = { ["en"] = { vim.fn.getcwd() .. "/false.ltex" } },
         settings = {
             ltex = {
                 enabled = { "latex", "tex", "bib", "markdown" },
```

However, there seems to be some bugs around ltex actually respecting the files contents. 

In short, I think you can remove the handlers from this repo (go to what the lsp-config PR is now). That makes ltex _work_ but does not handle it's local commands. Folks should use grammar-guard.nvim (after it is extended) to get the local commands.

**Comment by williamboman** [comment]
Thanks for the context @abalmos! Updated the server config to follow upstream changes. Once the lspconfig PR is merged we can remove the overrides and only provide `cmd` in this plugin
